common/main/en.xml

```diff
@@ -9443,6 +9443,8 @@ annotations.
 			<nameField type="given-informal">Neele</nameField>
 			<nameField type="given2">Eva Sophia</nameField>
 			<nameField type="surname">Meyer Wolf</nameField>
+			<nameField type="surname-prefix">van den</nameField>
```

Comment:
if both 'surname' and the pair 'surname-prefix' and 'surname-core' exist, they should be aligned. So this needs to change "Meyer Wolf" to "van den Wolf"

Reply:
We probably need a test for that also.